packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -915,11 +925,11 @@ class DataGrid extends React.Component {
     );
   }
 
-  renderCell(section, row, column) {
-    const { onCellSelect, defaultColumnWidth } = this.props;
-
+  renderCell(section, row, column, isFirstRow, isLastRow) {
+    const { onCellSelect, defaultColumnWidth, columnHighlightId } = this.props;
     const cell = (row.cells && row.cells.find(searchCell => searchCell.columnId === column.id)) || {};
     const cellKey = `${section.id}-${row.id}-${column.id}`;
+    const isHighlighted = (column.id === columnHighlightId);
```

Comment:
Can we rename `isHighlighted ` to `isColumnHighlighted` to be more specific to avoid confusion with other types of highlighting for row/cell/etc?

Reply:
[Renamed](https://github.com/cerner/terra-clinical/pull/796/commits/58bec54cb5437d28b3cc5f69cca242e6544f6f77#diff-412f754b40a1999c9f850f95d836fe5f21873db6a3feed84cbd813a65786e5ecR932)